stages/org.osbuild.erofs

```diff
@@ -84,6 +84,7 @@ def main(inputs, output_dir, options):
             method += f",{compression['level']}"
         cmd += ["-z", method]
 
+    options = options.get("options")
```

Comment:
Oops! Glad you caught that!

Reply:
Thanks! The mocked subprocess tests caught it :) It seems mkfs.erofs does not validate options, i.e.:
```
$ mkfs.erofs -E invalid-option /tmp/lala.img /tmp/foo
mkfs.erofs 1.6
Build completed.
```
works without an error :/

Reply:
> Thanks! The mocked subprocess tests caught it :) It seems mkfs.erofs does not validate options, i.e.:
> 
> ```
> $ mkfs.erofs -E invalid-option /tmp/lala.img /tmp/foo
> mkfs.erofs 1.6
> Build completed.
> ```
> 
> works without an error :/

1.7 already fixes that
https://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git/commit/?id=914c7f667377ad82c174265840bbf4a52b2dae6c

Reply:
Thanks for the fix @hsiangkao - that is amazing!

Reply:
FTR, 1.7 landed in Fedora 2 weeks ago, but our Fedora snapshot is too old.

https://bodhi.fedoraproject.org/updates/FEDORA-2023-ba67a1c746